.storybook/main.js

```diff
@@ -92,9 +92,15 @@ module.exports = {
         // Optimize and process SVGs as React elements for use in documentation
         test: /\.svg$/,
         use: '@svgr/webpack',
+      },
+      {
+        test: /\.(ts|tsx)$/,
+        loader: require.resolve('babel-loader'),
```

Comment:
Any reason this doesn't use the `use: 'babel-loader'` formatting of the other configurations?